scripts/build-types.ts

```diff
@@ -0,0 +1,24 @@
+import cue from 'cuelang-js';
+import { default as openapiTS } from 'openapi-typescript';
+
+import { promises as fs } from 'fs';
+
+const cueToTSTypes = async (input: string) => {
+	const cueresult = await cue('export', [input], { '--out': 'openapi' });
```

Comment:
Multiple words should use camelCase. I would call it `exportResult`

Reply:
Fixed in latest commit